problems/objects.js

```diff
@@ -187,15 +264,36 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+  let obj = countOccurance(str)
+  delete obj[" "]
+  return obj
+}
+
+
 
 /**
  * Takes in an array and returns the most common element.
  * @param {Array} array - array of elements
  * @returns {(number|string)} Most common element
  */
 
-const mostCommonElement = () => {};
+const mostCommonElement = (array) => {
+let count = {}
+for (let element of array){
+  count [element] = (count[element] || 0) +1
```

Comment:
Do you fully understand this line of code? Make sure that you do and could repeat it and explain in a job interview.

Reply:
Also please watch your spacing count[element]